cfme/physical/physical_server.py

```diff
@@ -68,58 +69,90 @@ def load_details(self, refresh=False):
             view.browser.refresh()
             view.flush_widget_cache()
 
-    def execute_button(self, button_group, button, handle_alert=False):
+    def execute_button(self, button, option, handle_alert=False):
```

Comment:
**REQUIRED**
This should be a private method too.

Reply:
ok